frontend/src/components/AddDatasetDrawer/AddDatasetDrawer.tsx

```diff
@@ -14,20 +14,20 @@ import './AddDatasetDrawer.css'
 
 const { ENV } = process.env
 const githubLink = SecureLink('https://github.com/bayesimpact/encompass', 'GitHub')
+const contactUsLink = SecureLink('mailto:encompass@bayesimpact.org?subject=Driving%Time%20Analysis', 'contact us')
 
 export let AddDatasetDrawer = withStore('selectedDataset', 'useCustomCountyUpload')(({ store }) =>
   <div className='AddDatasetDrawer'>
     <BackLink />
     <h2 className='Secondary'>Upload your data to explore</h2>
     <div className='ExplainerText'>
       <span className='MediumWeight'>
-        To analyze the accessibility of your own set of providers, facilities, or social services, you
-        will need to upload two separate CSV files:
-        <ul>
-          <li>List of service areas (county names)</li>
-          <li>List of locations for providers or services (latitude and longitude provided in separate columns)</li>
-        </ul>
-        For simplicity, this analysis will only produce results using straight-line distance. To run your own drive-time analysis, visit our {githubLink} page to learn how or contact us.
+      To analyze the accessibility of your own set of providers, facilities, or social services:
+      <ul>
+        <li>Isolate the population you want to analyze by selecting a state</li>
+        <li>Upload a CSV of locations representing providers or services. The header row must define "latitude" and "longitude" columns (and an optional "name" column).</li>
```

Comment:
This body of text and the closing span tag on L31 are out of alignment by one level of indentation.